dll/win32/shell32/dialogs/filedefext.cpp

```diff
@@ -1309,12 +1376,19 @@ CFileDefExt::CountFolderAndFiles(HWND hwndDlg, LPWSTR pwszBuf, UINT cchBufMax, D
     /* Find filename position */
     UINT cchBuf = wcslen(pwszBuf);
     WCHAR *pwszFilename = pwszBuf + cchBuf;
+    WCHAR PathBuffer[MAX_PATH] = L"";
+    WCHAR WorkBuffer[MAX_PATH] = L"";
+
     size_t cchFilenameMax = cchBufMax - cchBuf;
     if (!cchFilenameMax)
         return FALSE;
     *(pwszFilename++) = '\\';
     --cchFilenameMax;
 
+    // Store path without wildcard
+    ZeroMemory(PathBuffer, sizeof(PathBuffer));
```

Comment:
Why clear the buffer if you are immediately writing to it?

Reply:
Leftover of preliminary code. Removed.